jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -584,18 +600,18 @@ protected void runLoop() {
 
         // Update the frame buffer size from 2nd frame since the initial value
         // of frame buffer size from glfw maybe incorrect when HiDPI display is in use
-        if (framesAfterContextStarted < 2) {
-            framesAfterContextStarted++;
-            if (framesAfterContextStarted == 2) {
-                int[] width = new int[1];
-                int[] height = new int[1];
-                glfwGetFramebufferSize(window, width, height);
-
-                if (settings.getWidth() != width[0] || settings.getHeight() != height[0]) {
-                    listener.reshape(width[0], height[0]);
-                }
-            }
-        }
+        // if (framesAfterContextStarted < 2) {
+        //     framesAfterContextStarted++;
+        //     if (framesAfterContextStarted == 2) {
+        //         int[] width = new int[1];
```

Comment:
Unless there's a plausible reason for this code to be re-enabled in the future, I'd rather you deleted it. Commented-out code without any clear purpose adds to the burden of software maintenance.

Reply:
Yeah, the comment there says
```
        // Update the frame buffer size from 2nd frame since the initial value
        // of frame buffer size from glfw maybe incorrect when HiDPI display is in use
```
I didn't find this bug to be present, but we might run into it at some point, so i think it's better to leave the supposed fix commented out there.

Reply:
> I didn't find this bug to be present,

The code in question was added by @jmecn. Please talk to them and try to understand why they saw this bug and you don't.

Here's the relevant Forum/Hub topic: https://hub.jmonkeyengine.org/t/switching-to-lwjgl-3/41222

Reply:
I've read the forum post, now it is clear to me that this is a workaround for https://github.com/jMonkeyEngine/jmonkeyengine/issues/1751 ... however it means also that in the first two frames the framebuffer size will be wrong... it might cause unexpected results in user apps.
For now i've updated the PR to add it back, i'll see if i can come up with a different workaround

Reply:
I applied a different workaround that doesn't meddle with the initialization flow